code/game/objects/items/weapons/tanks/jetpack.dm

```diff
@@ -133,11 +133,6 @@
 	item_state = "jetpack-captain"
 	volume = 90
 	w_class = WEIGHT_CLASS_NORMAL
-	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF //steal objective items are hard to destroy.
-
-/obj/item/tank/jetpack/oxygen/captain/Initialize(mapload)
-	. = ..()
-	RegisterSignal(src, COMSIG_PARENT_QDELETING, PROC_REF(alert_admins_on_destroy))
```

Comment:
this was intentionaly removed because it's not longer available in the office yah?

Reply:
Correct. Jetpack is no longer an objective / in captains office, so this is not longer needed